File: tiendapome.frontend/src/app/models/parametro.ts
export class Parametro {      
      public _id: string;
      constructor (
            public Id:            number,
            public Descripcion:   string,
            public Vigente:       boolean,
            public Clave:         string,
            public Valor:         string,
            public Editable:      boolean
      ){}
}

File: tiendapome.backend/tiendapome.Entidades/Parametro.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace tiendapome.Entidades
{
    [JsonObject]
    public class Parametro : EntidadBaseParametrica
    {
        [JsonProperty("Clave")]
        public virtual string Clave { get; set; }

        [JsonProperty("Valor")]
        public virtual string Valor { get; set; }

        [JsonProperty("Editable")]
        public virtual Boolean Editable { get; set; }

        public Parametro() { }
    }
}
